tiles/src/main/java/com/protomaps/basemap/layers/Pois.java

```diff
@@ -323,7 +323,8 @@ public void processFeature(SourceFeature sf, FeatureCollector features) {
         } else if (kind.equals("forest") ||
           kind.equals("park") ||
           kind.equals("protected_area") ||
-          kind.equals("nature_reserve")) {
+          kind.equals("nature_reserve") ||
+          kind.equals("village_green")) {
```

Comment:
This could work out... but might need some adjustments. Generally village greens are pretty small, and the other features in this grouping can get much larger.

Most similar Tilezen section is here https://github.com/tilezen/vector-datasource/blob/master/yaml/pois.yaml#L143-L152, but there's some related clamping for `commons`.

Reply:
What adjustments are you thinking about?